app/angular/src/server/framework-preset-angular.ts

```diff
@@ -27,7 +27,7 @@ export function webpack(
         },
         {
           test: /[/\\]@angular[/\\]core[/\\].+\.js$/,
-          parser: { system: true },
+          parser: { system: false },
```

Comment:
@kroeder this change allows to hide these warnings
https://github.com/storybookjs/storybook/pull/15184#issuecomment-858704891

but I have no idea what these lines are for 😁